Testscripts/Linux/KVP-BasicChecks.sh

```diff
@@ -19,7 +19,7 @@
 }
 # Source constants file and initialize most common variables
 UtilsInit
-
+homeDir=$(pwd)
```

Comment:
Does it really need the $pwd  result?  what about to replace it with  $(dirname "$0"), just in case, all those dependencies ,like /${kvp_client}, are not under $pwd, but actually, they will always stick to the same folder as current KVP-BasicChecks.sh